nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/blob/ColdStorageHelper.java

```diff
@@ -74,6 +77,43 @@ public static DocumentModel moveContentToColdStorage(CoreSession session, Docume
         return session.saveDocument(documentModel);
     }
 
+    /**
+     * Retrieves the cold storage content associated with the document of the given {@link DocumentRef}.
+     * 
+     * @param session the core session
+     * @param documentRef the document reference
+     * @param numberOfDaysOfAvailability number of days that you want your cold storage content to be accessible after
+     *            restoring
+     * @apiNote This method will initiate a request restore, calling the {@link Blob#getStream()} during this process
+     *          doesn't mean you will get the blob content immediately.
+     * @return the updated document model if the retrieve succeeds
+     * @throws NuxeoException if there is no cold storage content associated with the given document, or if it is being
+     *             retrieved
+     */
```

Comment:
Maybe add a TODO here to explain that for now it only sets a boolean property but later on it will actually trigger the restore request to the cold storage?